src/FsSpreadsheet/Ranges/FsRangeBase.fs

```diff
@@ -78,7 +75,7 @@ type FsRangeBase (rangeAddress : FsRangeAddress, styleValue) =
     member self.Cells(cells : FsCellsCollection) = 
         cells.GetCells(self.RangeAddress.FirstAddress, self.RangeAddress.LastAddress)
        
```

Comment:
Is `CellsSelect` some kind of filter function? If yes pls rename. `Select` is a word used in xlsx context for any cells actually selected by the user in the xlsx file.

Reply:
![image](https://github.com/CSBiology/FsSpreadsheet/assets/39732517/5a9b63c8-f70d-49c3-baab-64b975dc762c)